Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.3.1'
+ruby '2.3.3'
 
-gem 'jekyll'
-gem 'html-proofer'
```

Comment:
I believe it's useful to specify versions in the Gemfile - see: https://everydayrails.com/2012/09/11/bundler-rails-specify-versions.html

Especially in a project where we're expecting people will install locally and might run `bundle update`.